manifests/m/Mayakron/AcrylicDNS/2.1.0/Mayakron.AcrylicDNS.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Mayakron.AcrylicDNS
+PackageVersion: 2.1.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://sourceforge.net/projects/acrylic/files/Acrylic/2.1.0/Acrylic.exe/download
+  InstallerSha256: 9565BAEFBAF938967D45B88889BE483AB67A56281771B36609F03B9F0253DB2F
```

```suggestion
  InstallerSha256: 9565BAEFBAF938967D45B88889BE483AB67A56281771B36609F03B9F0253DB2F
  Scope: machine
```
Installation Location is `C:\Program Files (x86)\Acrylic DNS Proxy`